.storybook/preview.js

```diff
@@ -38,4 +38,4 @@ export const parameters = {
   }
 };
 
-export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);
+export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);
```

Comment:
Can you revert this change? It looks like the new line was removed. I just want to try and keep the history tidy. Thanks!

Reply:
@mrtimbrook do you mean the new line at line 40, I've checked and it'd there? or do you mean there is one missing below `export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);`

Reply:
@mrtimbrook I'll PR the CORS workaround later on this evening

Reply:
@RobertByrnes Yeah, the one at the end of the file (after line 41).